src/layouts/components/Editable/AddSectionButton.tsx

```diff
@@ -20,6 +20,7 @@ export const AddSectionButton = ({
             isStretch
             isOpen={isOpen}
             rightIcon={undefined}
+            size="lg"
```

Comment:
assumed to be cleared w/ design

Reply:
Partially yes. Design mentioned that it should be the `lg` size of the `Button` prop in the design system. However, we need to use the Menu button to support a dropdown menu for the homepage. So design will follow-up separately with design system team to synchronise the size values for both the Button and Menu components.

For now, this is okay until design system updates to have the correct sizing, no changes will be needed from our end.